README.md

```diff
@@ -140,9 +142,15 @@ flags manually in whatever format they use.
 #### Rsync module
 
 If you set `rsyncSSHKeys`, an rsync module will be available at
-`ssh://mc-${name}@yourserver:${name}`. This rsync module has
+`ssh://mc-${name}@yourserver::mc-${name}`. This rsync module has
```

Comment:
Is the double colon intentional?

Reply:
From what I remember, yes, it didn't work for me with a single colon